### ISSUE TITLE
PF10 DataExporter: xlsx export is failing

### ISSUE THREAD

**Issue by Mag00n** [primefaces/primefaces#12703]
### Describe the bug

TLDR: Is there any known class loading issues / conflicts with poi 5.0 and pf10 / weblogic 12.2.1 ?

Hello, we recently upgraded from pf 6.3 to 10.0. Our P:dataexporter is failing with XLSX only (xls and pdf work fine).

We receive the following 2 errors when trying to export xlsx. The xmlOptions error comes first. If we return to the page and try again the shared strings table error comes.

javax.servlet.ServletException: org.apache.xmlbeans.XmlOptions.setUseDefaultNamespace(Z)Lorg/apache/xmlbeans/XmlOptions;

javax.servlet.ServletException: Could not initialize class org.apache.poi.xssf.model.SharedStringsTable

I have reviewed the maven dependency hierarchy and there are no conflicting version of poi coming from maven. xmlbeans 4.0.0 is coming from poi-ooxml 5.0.0 after excluding xml beans from weblogic bea directory.

Environment: 
Java 1.8
weblogic 12.2.1
pf / pfe 10

Relevant POM entries:

		<dependency>
			<groupId>org.primefaces</groupId>
			<artifactId>primefaces</artifactId>
			<version>10.0.0</version>
		</dependency>

		<dependency>
			<groupId>org.primefaces.extensions</groupId>
			<artifactId>primefaces-extensions</artifactId>
			<version>10.0.0</version>
		</dependency>

<dependency>
			<groupId>org.apache.poi</groupId>
			<artifactId>poi</artifactId>
			<version>5.0.0</version>
			<exclusions>
				<exclusion>
					<groupId>org.slf4j</groupId>
					<artifactId>jcl-over-slf4j</artifactId>
				</exclusion>				
			</exclusions>
		</dependency>

		<dependency>
			<groupId>org.apache.poi</groupId>
			<artifactId>poi-ooxml</artifactId>
			<version>5.0.0</version>
			<exclusions>
				<exclusion>
					<artifactId>stax-api</artifactId>
					<groupId>stax</groupId>
				</exclusion>			
			</exclusions>
		</dependency>

<dependency>
			<groupId>com.github.librepdf</groupId>
			<artifactId>openpdf</artifactId>
			<version>1.3.23</version>
			
		</dependency>
		
		<dependency>
			<groupId>com.lowagie</groupId>
			<artifactId>itext</artifactId>
			<version>2.1.7</version>
		</dependency>

<dependency>
			<groupId>com.oracle.weblogic</groupId>
			<artifactId>weblogic-server-pom</artifactId>
			<version>12.2.1-0-0</version>
			<type>pom</type>
			<scope>provided</scope>
			<exclusions>			
				<exclusion>
					<groupId>com.oracle.weblogic</groupId>
					<artifactId>com.bea.core.xml.xmlbeans</artifactId>
				</exclusion>
				<exclusion>
					<groupId>com.oracle.weblogic</groupId>
					<artifactId>
						com.bea.core.xml.beaxmlbeans
					</artifactId>
				</exclusion>
				<exclusion>
					<groupId>com.oracle.weblogic</groupId>
					<artifactId>
						com.bea.core.xquery.xmlbeans-interop_1.8.0.0
					</artifactId>
				</exclusion>

### Reproducer

export a p:datatable using p:dataExporter with xlsx file type

### Expected behavior

Exporting should be successful as it is with xls and pdf file type

### PrimeFaces edition

Community

### PrimeFaces version

10.0.0

### JSF implementation

Mojarra

### JSF version

2.1

### Java version

1.8

**Comment by tandraschko** [comment]
sorry but we dont support 10.0 anymore, its already over 3 years old.
check the docs for the right poi version

otherwise i can just suggest you to use 14.0

**Comment by Mag00n** [comment]
We are using the poi version indicated in the docs. And 14 is not compatible with java 1.8

**Comment by tandraschko** [comment]
there must be a xmlbeans conflict somehow
but i cant help further sorry, im sure its working fine on jetty e.g. and somehow related to your project/your application server

**Comment by melloware** [comment]
If you need Java 8 then use 13.0.10 version
